<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>361</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YTVideoDownloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { 
	background-color: rgb(238, 238, 238);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 20pt &quot;Segoe UI&quot;;
	color: rgb(85, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>YTVideoDownloader</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 10pt &quot;Segoe UI&quot;;
	color: rgb(18, 18, 16);
}</string>
     </property>
     <property name="text">
      <string>Video URL Link:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-width: 2px;
	border-radius: 10px;
	border-style: solid;
	border-color: rgb(47, 47, 42);
	font: 50 italic 10pt &quot;Sıtka Small&quot;;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 10pt &quot;Segoe UI&quot;;
	color: rgb(18, 18, 16);
}</string>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border-width: 2px;
	border-radius: 10px;
	border-style: solid;
	border-color: rgb(47, 47, 42);
	font: 50 italic 10pt &quot;Sıtka Small&quot;;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(244, 244, 244);
	border-width: 	2px;
	border-radius: 5px;
	border-style: solid;
	border-color: black;
}

QPushButton::hover
{
	background-color: rgb(217, 217, 217);
	border-width: 	1px;
	border-radius: 5px;
	border-style: solid;
	border-color: black;
}
QPushButton::pressed
{
	background-color: rgb(217, 217, 217);
	border-width: 	1px;
	border-radius: 5px;
	border-style: solid;
	border-color: black;
}</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(18, 18, 16);
}</string>
     </property>
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator::unchecked
{
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	border-width: 1px;
	border-style: solid;
	border-color: black;
	border-radius: 3px;
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}

QRadioButton::indicator::checked
{

	background-color: rgb(47, 47, 42);
	border-width: 1px;
	border-style: solid;
	border-color: black;
	border-radius: 3px;
}</string>
     </property>
     <property name="text">
      <string>MP4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton::indicator::unchecked
{
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	border-width: 1px;
	border-style: solid;
	border-color: black;
	border-radius: 3px;
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}

QRadioButton::indicator::checked
{

	background-color: rgb(47, 47, 42);
	border-width: 1px;
	border-style: solid;
	border-color: black;
	border-radius: 3px;
}</string>
     </property>
     <property name="text">
      <string>MP3</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
